app/hooks/useEmbedIntegrations.ts

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { IntegrationType } from "@shared/types";
+import Integration from "~/models/Integration";
+import Logger from "~/utils/Logger";
+import useStores from "./useStores";
+
+export default function useEmbedIntegrations() {
+  const { integrations } = useStores();
+  const [embedIntegrations, setEmbedIntegrations] = React.useState<
+    Integration<IntegrationType.Embed>[] | undefined
+  >();
+
+  React.useEffect(() => {
+    async function fetchEmbedIntegrations() {
+      try {
+        await integrations.fetchPage({
+          limit: 100,
+          type: IntegrationType.Embed,
+        });
+        setEmbedIntegrations(integrations.orderedData);
```

Comment:
I don't think there's any need for the local state in here

Reply:
Fixed!